src/components/contexts/error-handler.tsx

```diff
@@ -52,6 +56,24 @@ const ErrorHandler: React.FC = ({ children }) => {
     setError: handleError
   }), [ error ]);
 
+  /**
+   * Returns current time
+   * 
+   * @returns current time
+   */
+  const getTime = () => {
+    return moment().format("DD.MM.YYYY HH:mm:ss");
+  };
+  
+  /**
+     * Returns current window URL
+     * 
+     * @returns current window URL
+     */
```

Comment:
```suggestion
  /**
   * Returns current window URL
   * 
   * @returns current window URL
   */
```
Too many indentations in the doc